spec/lib/twilio_adapter_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe TwilioAdapter do
+  describe "#send_text" do
+    it "should send api request to twilio to send text" do
+      adapter = described_class.new
+      VCR.use_cassette("twilio_text") do
+        message = adapter.send_text('6178945641', 'testing a text')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.